incubator/vertx/backup.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I guess this is a debug file, would you remove it?

Reply:
You are absolutely right - removed